### ISSUE TITLE
amazfiz freezes with this watchface

### ISSUE THREAD

**Issue by jdelacasa** [Klaus3d3/xDrip-Watchface#5]
amazfiz freezes with this watchface, After a while, only the clock restarts. You have to desistall it so that the relog will work again

**Comment by roco5** [comment]
I have the frozen watchface as well?  It was working fine, but now I get this error:  Unfortunately xdrip +watchface +Service has stopped

**Comment by krixoko** [comment]
have the same issue, worked well before.

Will try to reinstall.

Thank you Klaus for this amazing work.

**Comment by watchdog7** [comment]
Weird, i got the same issue.

I even did a Factory Reset, but it didn't help. 

Anyone found a solution?

**Comment by vebaba** [comment]
Just change xdrip watchface language from English to any other. In reality, you will have to read month and day names in that other language. It seems there was a problem with months list in English...

**Comment by watchdog7** [comment]
Do you mean Settings > xDrip+ Display Settings > Choose a specific Language?

Or is there a specific setting for Watchfaces?

**Comment by vebaba** [comment]
Cheers!
Additional explanation: in time you change/choose xdrip+ watchface, there is Settings. After choosing Settings, that there is an option to choose language for xdrip+ watchface. There, you can choose anything but English (I believe it will work again in January :) )

**Comment by Klaus3d3** [comment]
Yes, maybe you noticed there is also a widget coming with the watchface.
